aerogear/mobile-security-service-operator
# fix(NamespaceCheck) : Fix namespace check for production and local env

## Motivation
- https://issues.jboss.org/browse/AEROGEAR-9233
- https://issues.jboss.org/browse/AEROGEAR-8809

## What
- Remove skip from CR 
- Use instance namespace
- Check if it is running out of the cluster (unit test/local env) in the workarround valid namespace check

## Why
- Fix/improvements/Security

## How
- Check that it is a local env or a unit test and then return a default value for the Operator Namespace
- The isValidAppNamespace need to use the instance.Namespace
- Remove skip from CR

## Verification Steps
Test with the image: docker.io/cmacedo/mobile-security-service-operator:FIX-NAMESPACE-CHECK
 
1. Check that the operator was created with success in the cluster (make create-all)
2. Check the features to create and delete app ( Check ConfigMap and Service )
3. Check if the project will run locally to debug and etc with the command `make run-local` 
4. Check if it is passing in the CI

## Checklist:


## Progress


## Additional Notes
